tests/unit/commands/_utils/custom_options/test_hook_package_id_option.py

```diff
@@ -13,6 +13,12 @@ def setUp(self):
         self.name = "hook-package-id"
         self.opt = "--hook-package-id"
         self.terraform = "terraform"
+        self.invalid_coexist_options = ["t", "template", "template-file", "parameters-override"]
+        self.metadata_path = "path/metadata.json"
+        self.cwd_path = "path/current"
+
+        self.iac_hook_wrapper_instance_mock = MagicMock()
+        self.iac_hook_wrapper_instance_mock.prepare.return_value = self.metadata_path
```

Comment:
This file has a bunch of changed lines. The bulk of the changes is to move a wrapper mock and some setup to the `setUp` class to reduce repeat code.